composer.json

```diff
@@ -61,6 +61,8 @@
     },
     "minimum-stability": "dev",
     "require": {
+        "ext-fileinfo": "*",
+        "ext-simplexml": "*",
```

Comment:
`ext-simplexml` is required by `qti-sdk`, [here](https://github.com/oat-sa/qti-sdk/blob/master/composer.json#L33), and this repository wouldn't work without `qti-sdk`.
Why did you need to add `ext-fileinfo` requirement?

Reply:
previously i used `mime_content_type` for for determining asset type. but after this behavior was simplified. And now `ext-fileinfo` can be deleted